.env.development.example

```diff
@@ -1,5 +1,10 @@
-REDIS_URL=redis://localhost:6379/0
-DATABASE_URL=postgres://localhost:5432/app_perf?encoding=utf8&pool=5&timeout=5000
```

Comment:
Update to have these env as well as redis and postgres gem handles the check of env being available